saleor/product/models.py

```diff
@@ -898,7 +895,36 @@ def get_ordering_queryset(self):
         return self.product.collectionproduct.all()
 
 
-class Collection(SeoModel, ModelWithMetadata, PublishableModel):
+class CollectionsQueryset(models.QuerySet):
```

Comment:
Let's use the same method link in ProdcutsQueryset. If you want to implement a new convention for you should refactor all old approach.

Reply:
The changes are already implemented, I didn't commit changes because of extend this PR with removing homepage collection.